api/provisioner/provisioner.go

```diff
@@ -209,8 +209,9 @@ func (st *State) prepareOrGetContainerInterfaceInfo(
 	if err := result.Results[0].Error; err != nil {
 		return nil, err
 	}
-	ifaceInfo := make([]network.InterfaceInfo, len(result.Results[0].Config))
-	for i, cfg := range result.Results[0].Config {
+	machineConf := result.Results[0]
```

Comment:
This was originally because we were using more attributes of this, could probably just revert this